api/v1alpha1/fenceagentsremediation_types.go

```diff
@@ -31,21 +31,26 @@ const (
 	// FenceAgentActionSucceededType is the condition type used to signal whether the Fence Agent action was succeeded successfully or not
 	FenceAgentActionSucceededType = "FenceAgentActionSucceeded"
 	// condition messages
-	RemediationStartedConditionMessage  = "FAR CR was found, the CR name matches one of the cluster nodes, and a finalizer was set"
-	FenceAgentSucceededConditionMessage = "FAR taint was added, fence agent command has been created and executed successfully"
-	RemediationFinishedConditionMessage = "The unhealthy node was fully remediated (it was tainted, fenced using FA and all the node resources have been deleted)"
+	RemediationInterruptedConditionMessage          = "FAR CR name doesn't match a node name or Node Healthcheck timeout annotation has been set"
```

Comment:
one reason for actual 2 reasons isn't that helpful for users, is it?

Reply:
Correct, I neglected the value of separating the messages.